Gems/EMotionFX/Code/Source/Integration/System/AnimationModule.cpp

```diff
@@ -51,6 +51,7 @@ namespace EMotionFX
             : public CryHooksModule
         {
         public:
+            AZ_CLASS_ALLOCATOR(EMotionFXIntegrationModule, AZ::SystemAllocator)
```

Comment:
General question, why do most pass a `0` as third parameter, but some don't?

Reply:
The `AZ_CLASS_ALLOCATOR` macro used to take 3 parameters, but due to other changes the 3rd parameter stopped being used. It was made an optional parameter in https://github.com/o3de/o3de/commit/2e59035cfa79fe959acc272bbefb61ab09d23cd4 (line 246 of Memory.h)

Reply:
If its not used, isn't it worth it to go back and remove it from the other files in this PR? I can see copy/paste situations where the unused parameter keeps getting perpetuated, and these are all new changes.

Reply:
Yeah, I address with @spham-amzn, if new version of the macro are being added, then they should all be added in a consistent manner.

Reply:
I went ahead and removed the third parameter from all the uses that are being added by this PR.